python/ray/autoscaler/_private/providers.py

```diff
@@ -6,6 +6,8 @@
 
 import yaml
 
+import ray
```

Comment:
why is this required?

Reply:
For `ray.__version__` and `ray.__commit__`